s3parq/publish_parq.py

```diff
@@ -164,7 +165,17 @@ def log_size_estimate(num_bytes):
     return tuple(sized_frames)
 
 
-def publish(bucket: str, key: str, partitions: iter, dataframe: pd.DataFrame) -> None:
+def publish(bucket: str, key: str, partitions: iter, dataframe: pd.DataFrame, redshift_params = None) -> None:
+    '''Redshift Params:
```

Comment:
Can this be adjusted to standard doc strings? Example : https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html . fetch_parq has some simple examples

Reply:
Done